src/index.html

```diff
@@ -7,11 +7,17 @@
     <title>Document</title>
   </head>
   <body class="d-flex flex-column h-100">
-    <header></header>
-    <main>
-      <h1 class="">Hello World!</h1>
-      <button class="btn btn-primary">Clic</button>
-    </main>
+    <header>
+      <nav class="navbar navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">
+            <i class="fab fa-product-hunt d-inline-block"> </i>
+            Pokescripts
+          </a>
+        </div>
+      </nav>
+    </header>
```

Comment:
I think is a good idea to add some options to the navbar like the example.

![example](https://user-images.githubusercontent.com/7466903/123557332-ac3eee80-d790-11eb-91c8-9f8a6a38a4ff.png)

Reply:
Ready you can check this

Reply:
Well done!! 👍🏽👍🏽